docs/admin/gradle_upgrade.md

```diff
@@ -0,0 +1,8 @@
+## How to upgrade gradle
+
+- Gradle distributions can be found at: https://services.gradle.org/distributions/ (we use a '-all' version)
+- Update the gradle version in: https://github.com/triplea-game/triplea/blob/master/gradle/wrapper/gradle-wrapper.properties
+- Execute: `./gradlew wrapper`
+- Change the gradle zip back to '-all' from '-bin' in gradle-wrapper.properties
```

Comment:
Running `./gradlew wrapper --gradle-version=4.6 --distribution-type=all` would probably be simpler.